### ISSUE TITLE
Support for default field tag

### ISSUE THREAD

**Issue by n0madic** [alexflint/go-arg#68]
Like this:
```go
var args struct {
    Config string `arg:"-c" default:"config.ini"`
}
```

**Comment by alexflint** [comment]
Why not use the existing mechanism for default values, ie https://github.com/alexflint/go-arg#default-values ?

The advantage of the existing mechanism is that the default values are parsed and type-checked by the compiler, whereas using a `default` tag would have to be parsed and type-checked at runtime.

**Comment by n0madic** [comment]
I use it, but IMHO is not so clear when the structure of the arguments is described in one place, and its defaults in another..

I would like to have both options.

**Comment by wrfly** [comment]
I'll make a PR for this feature using this lib https://github.com/wrfly/ecp
@n0madic I think this can satisfy your requirement.

**Comment by n0madic** [comment]
I would like to have it bundled without unnecessary dependencies, but if not, it will be so ...

**Comment by wrfly** [comment]
I wrote another lib called `gua`, maybe you are interested... https://github.com/wrfly/gua

**Comment by alexflint** [comment]
Fixed in #91
